pom.xml

```diff
@@ -72,7 +72,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <declared.ozone.version>${ozone.version}</declared.ozone.version>
 
     <!-- Apache Ratis version -->
-    <ratis.version>2.1.0</ratis.version>
+    <ratis.version>2.1.0-03f3b68-SNAPSHOT</ratis.version>
```

Comment:
Sorry, but this seems to be wrong.  Ratis 2.1.0 is already released, we cannot have a new snapshot version for it.  I think we should go for a new 2.1.1 release with RATIS-1384 (and RATIS-1386).